File: ADS 2024.1/EDI/listas_fabio/lista01/q16.c
#include<stdio.h>
int main(){
    float size;
    float area;

    
    printf("Enter the side value: ");
    scanf("%f", &size);

    

    area = size * size;

    printf("The area of the square with size %.2f is: %.2f", size, area);





    return 0;
}

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q16.js
import { question } from "readline-sync";
function main(){
    //Input
    const side_square = parseFloat(question("Type the value of the square of side:"))

    //Processing
    const area = side_square ** 2
    const rounded_area = area.toFixed(2)

    //Output
    console.log(`The value of the area of the square with length of the side ${side_square} is: ${rounded_area} a.u`)

}

main()